src/Model.php

```diff
@@ -577,25 +590,28 @@ public function allFields()
         return $this;
     }
 
-    private function checkOnlyFieldsField(string $field)
+    private function assertOnlyFieldsField(string $fieldName)
```

Comment:
We should probably change to `assertAccessibleField` or similar as naming now does not match the logic in other places.
onlyFieldsField is strictly a field in `only_fields` irrelevant if it is `system` or not